### ISSUE TITLE
Add "force" option to save or update

### ISSUE THREAD

**Issue by ralfthewise** [tools/godep#455]
Often times after merging branches my "vendor" directory is left in an inconsistent state and I just want to re-vendor.  However, doing a `godep save` or `godep update` ends up being a no-op because it seems like godep assumes there are no changes based on the SHAs in `Godeps.json`.  It would be nice to be able to do `godep save -f` or some such that would always copy dependencies to "vendor", even if it thinks things are already up to date.

**Comment by frankcarey** [comment]
Yeah, I second this.. also seemed to happen if I manually deleted my vendor folder that it wasn't being recreated.

**Comment by freeformz** [comment]
Closing issues before archiving. Thank You\!
